gustaf/io/npz.py

```diff
@@ -0,0 +1,120 @@
+"""gustaf/gustaf/io/npz.py
+
+io functions for numpy.savez.
+"""
+
+import numpy as np
+
+from gustaf.vertices import Vertices
+from gustaf.edges import Edges
+from gustaf.faces import Faces
+from gustaf.volumes import Volumes
+
+# tuple instead of dict for correct iteration order
+_types_and_arrays = (
```

Comment:
could you please make this a dict?

Reply:
It can't be a `dict` because the order needs to be maintained. I could make it an `OrderedDict` though.